jest.config.js

```diff
@@ -23,10 +23,6 @@ module.exports = {
     "packages/remix-server-runtime",
     "packages/remix-testing",
   ],
-  watchPlugins: [
-    require.resolve("jest-watch-select-projects"),
-    require.resolve("jest-watch-typeahead/filename"),
-    require.resolve("jest-watch-typeahead/testname"),
-  ],
-  reporters: ["default"],
+  watchPlugins: ["select-projects", "typeahead/filename", "typeahead/testname"],
```

Comment:
any particular reason these are not `require.resolve` anymore?

Reply:
Just because the docs use string tbh
Is `require.resolve` preferred?

Reply:
it's a bit faster as Jest doesn't have to do its own resolution

Reply:
Reverted the change